pkg/azurefile/version_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azurefile
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"sigs.k8s.io/yaml"
+)
+
+func TestGetVersion(t *testing.T) {
+	get := GetVersion()
+
+	want := VersionInfo{
+		DriverName:    "N/A",
+		DriverVersion: "N/A",
+		GitCommit:     "N/A",
+		BuildDate:     "N/A",
+		GoVersion:     runtime.Version(),
+		Compiler:      runtime.Compiler,
+		Platform:      fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
+	}
+	if get == want {
+		t.Log("Got as expected")
+	} else {
+		t.Errorf("Unexpected Versions: Please check the requirements")
+	}
+
+}
+
+func TestGetVersionYAML(t *testing.T) {
+	_, get := GetVersionYAML()
```

Comment:
it's actually should be like this:
```
yaml, err := GetVersionYAML()
```
and you should compare the `yaml` with expected string and also check there is no `err`

Reply:
alright